python/flexflow/keras/models/base_model.py

```diff
@@ -18,6 +18,7 @@
 
 from .tensor import Tensor
 from flexflow.keras.layers import Conv2D, Pooling2D, Flatten, Dense, Activation, Concatenate, Add, Subtract, Multiply, Dropout, BatchNormalization, Embedding, Reshape
+from flexflow.keras.backend import BatchMatmul
```

Comment:
Does `BatchMatmul` exist in `keras.backend`?

Reply:
No. Need to rename it to `batch_dot`